src/System.Management.Automation/DscSupport/CimDSCParser.cs

```diff
@@ -1996,7 +1996,7 @@ private static ParseError[] CheckMandatoryPropertiesPresent(DynamicKeywordStatem
                     {
                         foreach (var psModuleInfo in moduleInfos)
                         {
-                            if ((moduleToImport.Guid.HasValue && moduleToImport.Guid.Equals(psModuleInfo.Guid)) ||
```

Comment:
Previously, if `moduleToImport.Guid` was `null` and `psModuleInfo.Guid` was `Guid.Empty`, this condition would be false. With the change it will be true. On that basis that this is a possible break rather than a style change, I'm going to close this PR